### PR TITLE
Updated Lessc stream and made less a peer dependency

### PR DESCRIPTION
**Pull request by rkesters**
The version of lessc was locked on an older version by lessc-stream, I got lessc-stream to move the lessc to a peer dependency. This PR updates to the new lessc-stream and make lessc a peer dep for lessc-glob.

@sag1v

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,8 +29,14 @@
...
     "chalk": "^1.1.3",
-    "less-css-stream": "^1.0.0"
+    "cpx": "^1.5.0",
+    "less-css-stream": "^2.0.0"
+  },
+  "peerDependencies": {
+    "less": ">=1.0.0"
+  },
+  "devDependencies": {
+    "less": "^3.10.3"
```
Comment (sag1v) [inline package.json]: Why is `less` is showing up twice?

**Comment by sag1v** [comment]
Hi, thanks for the PR. What was the initial issue?

**Comment by rkesters** [comment]
> Hi, thanks for the PR. What was the initial issue?

The issues is that the version of lessc is old and required both less-css-stream and lessc-glob to be re-published each time. I made a change to less-css-stream to make lessc a peer dependency, updated this package to also have lessc as a peer dependency. This allows the user of this package to select the version of lessc they want to use.

TL;DR The issue was that lessc was old and locked.

File: package.json (hunk 1 above)
Comment (rkesters) [reply package.json]: I'll remove the devDependencies , since there are no tests

**Comment by sag1v** [comment]
Great, thanks
